<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dailymoodClass</class>
 <widget class="QMainWindow" name="dailymoodClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dailymood</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0" colspan="3">
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TODOS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="updateMoodBtn">
       <property name="text">
        <string>UPDATE MOOD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="addTodoButton">
       <property name="text">
        <string>ADD TODO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="todosScroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>429</width>
       <height>229</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="moodScroll"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dailymood.qrc"/>
 </resources>
 <connections/>
</ui>
